openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -92,3 +139,66 @@ def change_activation_status(self, activation_status, ids):
             return Response(status=status.HTTP_200_OK)
 
         return Response(status=status.HTTP_404_NOT_FOUND)
+
+    def create_account_creation_form(self, request):
+        return AccountCreationForm(
+            data={
+                'allow_account_creation': '{allow_account_creation}'.format(allow_account_creation=True),
+                'username': '{username}'.format(username=request.data['username']),
+                'email': '{email}'.format(email=request.data['email']),
+                'password': '{password}'.format(password=uuid.uuid4().hex[:8]),
+                'name': '{name}'.format(name=request.data['username']),
+            },
+            tos_required=False
+        )
+
+    def create_new_user(self, request):
+        (user, user_profile, _) = do_create_account(self.create_account_creation_form(request))
+        user_profile.employee_id = request.data['employee_id']
+        user_profile.language = request.data['language']
+        user_profile.save()
+        self.specify_user_role(user, request.data.get("role"))
+        user.save()
+        protocol = 'https' if get_current_request().is_secure() else 'http'
```

Comment:
can we use request.scheme()? 
https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpRequest.scheme